### PR TITLE
Fix `manifest.json` indentation for some complex APIs

### PR DESCRIPTION
**Pull request by steff456**
Part of https://github.com/stdlib-js/todo/issues/1648

## Description

> What is the purpose of this pull request?

-   Fixes the indentation of some `manifest.json` files for the complex APIs

## Related Issues

> Does this pull request have any related issues?

This pull request:

-   part of https://github.com/stdlib-js/todo/issues/1648

## Questions

> Any questions for reviewers of this pull request?

No.

## Other

> Any other information relevant to this pull request? This may include screenshots, references, and/or implementation notes.

No.

## Checklist

> Please ensure the following tasks are completed before submitting this pull request.

-   [x] Read, understood, and followed the [contributing guidelines][contributing].

* * *

@stdlib-js/reviewers

[contributing]: https://github.com/stdlib-js/stdlib/blob/develop/CONTRIBUTING.md

### CODE REVIEW

**Review by kgryte — CHANGES_REQUESTED** [review]
Thanks, @steff456! Can you ensure that the `manifest.json` files have a trailing newline per EditorConfig [settings](https://github.com/stdlib-js/stdlib/blob/d33368119f5cd6d037eed7e1d9f24be3368832a9/.editorconfig#L29)?

**Comment by steff456** [comment]
Sure!!

File: lib/node_modules/@stdlib/math/base/special/cceil/manifest.json [hunk 1]
```diff
@@ -1,79 +1,78 @@
...
+      "libraries": [],
+      "libpath": [],
+      "dependencies": [
+        "@stdlib/complex/float64",
+        "@stdlib/complex/reim",
+        "@stdlib/math/base/special/ceil"
+      ]
+    }
+  ]
+}
```
Comment (kgryte) [inline lib/node_modules/@stdlib/math/base/special/cceil/manifest.json]:
```suggestion
}

```

**Review by kgryte — APPROVED** [review]
LGTM. Thanks, @steff456!

**Pull request merged**